yogstation/code/modules/mob/living/carbon/human/species_types/preternis/power_suck.dm

```diff
@@ -0,0 +1,186 @@
+/mob/living/carbon/AltClickOn(atom/A)
+	dna?.species.spec_AltClickOn(A,src)
```

Comment:
This completely overrides all altclick events. It should not completely override all altclick events.

Reply:
not my fault

Reply:
this fixes the bug that lets players rotate glass and pull items without a closet

Reply:
Don't fix bugs in a feature PR unless you created the bug in the same PR